cmd/srcd/cmd/defered.go

```diff
@@ -24,11 +25,27 @@ type Defered struct {
 	Spinner bool
 	// SpinnerInterval allows to change speed of spinner (200ms by default)
 	SpinnerInterval time.Duration
+
+	// logger is the go-log DefaultLogger. Can be changed for tests
+	logger log.Logger
+	// logWriter is the io.Writer where logger sends the output. By default,
+	// os.Stderr. Can be changed for tests
+	logWriter io.Writer
+	// forceIsTerminal forces spinner to be printed without checking if the
+	// output is a terminal. For tests
+	forceIsTerminal bool
```

Comment:
maybe we can rename it to `isTerminal` and set in the constructor.
For the tests you can do `d := newDefered(); d.isTerminal = true/false` or create this struct without a constructor.
Imo it will be cleaner code.

Reply:
Done in a338701